lib/sprite.dart

```diff
@@ -1,20 +1,21 @@
 import 'dart:ui';
 
+import 'extensions/offset.dart';
 import 'extensions/vector2.dart';
 import 'palette.dart';
 
 class Sprite {
   Paint paint = BasicPalette.white.paint;
   Image image;
-  Rect src;
+  Rect bounds;
 
   Sprite(
     this.image, {
     Vector2 position,
     Vector2 size,
```

Comment:
Add @required here?

Reply:
@spydon I totally 100% thought you were talking about SpriteComponent! only now I realized it was the actual Sprite class haha. sorry about that. I think this for sure could have a 0,0 default. then it doesn't need to be required. I was thinking spriteComponent.position might not be as clear with a default value but this one for sure can have one. In fact should be easy to create a sprite with no params other than the image: it should default to the whole image.